common/persistence/visibility/store/elasticsearch/client/config.go

```diff
@@ -80,12 +81,18 @@ type (
 // GetVisibilityIndex return visibility index name from Elasticsearch config or empty string if it is not defined.
 func (cfg *Config) GetVisibilityIndex() string {
 	if cfg == nil {
-		// Empty string is be used as default index name when Elasticsearch is not configured.
```

Comment:
It was important comment. Why did you remove it? It had grammar errors though.

Reply:
I will add this back. But how is it supposed to work if user does not specify index name?